cli/bpmetadata/proto/bpmetadata_ui_ext.proto

```diff
@@ -164,7 +164,7 @@ message GCEExternalIPExtension {
   bool not_configurable = 3; // @gotags: json:"notConfigurable,omitempty" yaml:"notConfigurable,omitempty"
 
   // Flag to denote if static IPs are allowed for the external IP.
-  bool allow_static_ips = 4; // @gotags: json:"allowStaticIPs,omitempty" yaml:"allowStaticIPs,omitempty"
+  bool allow_static_ips = 4 [json_name="allowStaticIPs"]; // @gotags: json:"allowStaticIPs,omitempty" yaml:"allowStaticIPs,omitempty"
```

Comment:
```suggestion
  bool allow_static_ips = 4; // @gotags: json:"allowStaticIps,omitempty" yaml:"allowStaticIps,omitempty"
```

Reply:
I think we just need to rename `allowStaticIPs` to `allowStaticIps` and that should fix the issue since that's how schemas were defined previously. Ref: go/allowstaticips-ref

Reply:
Done